src/Test/UI/AddQueueButton/AddQueueButton.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import styles from './AddQueueButton.module.css';
+import AddIcon from '@mui/icons-material/Add';
+function AddQueueButton() {
+    return (
+        <>
+            <a href="" className={styles.AddQueueButton}> <span className={styles.AddIcon}> <AddIcon fontSize="large" /> &nbsp; </span> Add Queue </a>
```

Comment:
It will be better to use the <Link> tag from react-router-dom or just a button rather than <a> in react as it is not redirecting to outside of our webpp.